examples/prod_failures.rb

```diff
@@ -24,3 +24,4 @@ def main
 end
 
 main if __FILE__ == $PROGRAM_NAME
+aaa    =   "dfdfdfdf"
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `aaa`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)